athena-oracle/src/main/java/com/amazonaws/athena/connectors/oracle/OracleJdbcConnectionFactory.java

```diff
@@ -37,14 +37,11 @@
 
 public class OracleJdbcConnectionFactory extends GenericJdbcConnectionFactory
 {
-    public static final String IS_FIPS_ENABLED = "is_FIPS_Enabled";
+    public static final String IS_FIPS_ENABLED = "is_fips_enabled";
```

Comment:
is this going to cause any issues for Cxs who are already using the previous version??

Reply:
actually that ignores case is for the `true` or `false`. This might be an issue

Reply:
so this is not entirely accurate; when we get the env variable we are not ingoring case sensitivity; only ignoring casing for the value of the variable env; not the name of the variable env itself.

Reply:
so if previous Cxs are using `is_FIPS_Enabled` in env variable; then this change will break it. So best way to do this is to check for both and add a warning if Cx is using `is_FIPS_Enabled` to migrate to use the new variable env.

Reply:
Resolved in PR #2470